### PR TITLE
Solved Issues

### PR DESCRIPTION
**Pull request by asimism**
1:- Left sidebar offline cameras toggle button
 2:- Switch between cameras list styled
 3:- Sharing tab mobile view styled
 4:- PTZ control issues solved

### CODE REVIEW

**Comment by asimism** [comment]
@azharmalik3 please merge and deploy this pull request, thanks in advance.

**Comment by azharmalik3** [comment]
@asimism please fix small silly hound issues.

File: app/assets/javascripts/sidebar.js.coffee [hunk 1]
```diff
@@ -30,6 +30,21 @@ renderSidebar = (cameras) ->
   $(".sidebar-cameras-list").html(sidebar)
   hideOfflineCameras()
 
+showOfflineButton = ->
+  a = $('.sub-menu.sidebar-cameras-list .sidebar-offline').length
```
Comment (azharmalik3) [inline app/assets/javascripts/sidebar.js.coffee]: Use reasonable variable name like `offline_cameras`.

File: app/assets/javascripts/sidebar.js.coffee [hunk 2]
```diff
@@ -30,6 +30,21 @@ renderSidebar = (cameras) ->
...
+  if a > 0
+    $('#offlineBtn').show()
+  else
+    $('#offlineBtn').hide()
+  $('#offlineBtn').on 'click', (event) ->
+    $(this).toggleClass('active')
+    $('.sub-menu').toggleClass('cam-active')
+    text = $(this).text()
+    if text == 'Hide Offline'
```
Comment (azharmalik3) [inline app/assets/javascripts/sidebar.js.coffee]: use `is` instead `==` sign.

**Comment by asimism** [comment]
@azharmalik3 Cleaned important Hound issues, whatever is remaining will remain there so please merge this pull request.

**Pull request merged**